assets/sidecar/developer-hub-app-config-patches.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+data:
+  RHDH_TOKEN: ase64-encoded-token-generated-from-openssl-or-similar-program
+kind: Secret
+metadata:
+  name: ai-rh-developer-hub-env
+  namespace: ai-rhdh
+type: Opaque
+
+---
+apiVersion: v1
+data:
```

Comment:
Is this configmap for testing, or part of what will be deployed?

IMO - I think it's fine for testing, but for production use cases, we'll probably want to make sure the fields we're adding merge fine with the user's pre-existing app-config.yamls (if there's any overlap)

Reply:
yeah I consider this whole `sidecar` folder for dev testing, and never beyond that, and a reference starting point for @thepetk to use gitops, kustomize, etc. for the production use case 

I thought from one of the recent video conf's @thepetk was fine with that

I should probably update the README with something to that affect i.e. it is just a ref for dev testing, and assumes for now you've used the ai-rhdh-installer

that said, once @thepetk 's work is done for the rolling demo, I am more inclined to have this repo simply point to that for setting up the side car, stand alone bridge pod, etc.

WDYT @thepetk @johnmcollier